.github/workflows/pull-request.yml

```diff
@@ -8,7 +8,11 @@ jobs:
   build-app:
     runs-on: ubuntu-latest
     steps:
-    - name: Check out the repo
-      uses: actions/checkout@v2
-    - name: Build docker image
-      run: docker build $DIRECTORY
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
```

Comment:
not sure if we want to run npm commands here or in the docker container?

Reply:
would be better in docker ideally, but can replace later